packages/libcacard.rb

```diff
@@ -0,0 +1,41 @@
+# Adapted from Arch Linux libcacard PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/libcacard/trunk/PKGBUILD
+
+require 'package'
+
+class Libcacard < Package
+  description 'Common Access Card library thar provides emulation of smart cards to a virtual card reader running in a guest virtual machine'
```

Comment:
`description 'Common Access Card library that`?  We want to avoid pirate talk. :)

Reply:
From copy... https://github.com/archlinux/svntogit-packages/raw/packages/libcacard/trunk/PKGBUILD

Fixing...